doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -4324,27 +4324,27 @@ In addition to the known issues listed here, see also :ref:`softdevice_controlle
 DRGN-23776: Sending CIS packets with invalid MIC
   When the CIS central running on an nRF5340 device is sending encrypted ISO packets, the MIC may be invalid.
 
-.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0
+.. rst-class:: v2-7-0 v2-6-3 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0
 
 DRGN-22443: A rare assert when disabling a periodic advertising set with responses
   In some rare cases, the controller can assert when disabling a periodic advertising set with responses.
 
-.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-7-0 v2-6-3 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
 
 DRGN-22441: The length byte of the HCI packet could be incorrect
   This could happen when the packet contains an LE BIG Sync Established event or LE BIG Complete event with status not equal to success.
 
-.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0
+.. rst-class:: v2-7-0 v2-6-3 v2-6-2 v2-6-1 v2-6-0
 
 DRGN-22652: Assert when receiving on the S8 Coded PHY
   This could happen in a connection where link peer device is transmitting on S8 Coded PHY.
```

Comment:
DRGN-22652 is fixed in NCS 2.6.3. DRGN-24327 also gives the same issue.

Workaround: Disable CONFIG_BT_CTLR_DATA_LENGTH and CONFIG_BT_DATA_LEN_UPDATE

Reply:
Discussed offline. Adding DRGN-24327 is not required.